channels/co.m3u

```diff
@@ -72,8 +76,9 @@ https://liveingesta118.cdnmedia.tv/teleantioquialive/smil:dvrlive.smil/playlist.
 #EXTINF:-1 tvg-id="Teleantioquia2.co" tvg-country="CO" tvg-language="Spanish" tvg-logo="" group-title="General",Teleantioquia 2 (360p) [Not 24/7]
 #EXTVLCOPT:http-referrer=https://player.instantvideocloud.net/
 https://liveingesta118.cdnmedia.tv/teleantioquialive/smil:dvrlive2.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="Telecafe.co" tvg-country="CO" tvg-language="Spanish" tvg-logo="http://cdn-std-1.sibasa.netdna-cdn.com/co/b3aef974216995f45cbb271e6be36964.jpg" group-title="General",Telecafé [Offline]
-http://38.131.11.9:1080/play/a0ct
+#EXTINF:-1 tvg-id="Telecafe.co" tvg-country="CO" tvg-language="Spanish" tvg-logo="http://cdn-std-1.sibasa.netdna-cdn.com/co/b3aef974216995f45cbb271e6be36964.jpg" group-title="General",Telecafé [Not 24/7]
+#EXTVLCOPT:http-referrer=https://player.instantvideocloud.net/
+https://liveingesta118.cdnmedia.tv/telecafelive/smil:dvrlive.smil/playlist.m3u8
```

Comment:
Can't confirm either

Reply:
https://telecafe.gov.co/en-vivo/

cant play it outside ..  referer on or with user agent

https://liveingesta118.cdnmedia.tv//telecafelive/smil:dvrlive.smil/playlist.m3u8?DVR
https://liveingesta118.cdnmedia.tv/telecafelive/dvrlive-900/chunklist.m3u8?DVR=
https://liveingesta118.cdnmedia.tv/telecafelive/dvrlive-1500/chunklist.m3u8?DVR=

 i get 403 on access 

it request 2 times 1x(playing) and 1x(presence)
https://rt.cdnmedia.tv:4001/
before getting chunk file and .ts

Reply:
yep, referer to https://player.instantvideocloud.net/ like other channels in this server.

Reply:
Don't ask me how, but channels began to respond again on my side. 403 Forbidden however when trying to read ts files, i'm looking at it.